### DISCUSSION TITLE
no winning plots found in over 4 months

### DISCUSSION THREAD

**Discussion by baerrs** [Chia-Network/chia-blockchain#12599] [Support]
I've not won a plot in over 4 months.  My last win was 4/12/2022.
The guest-o-mate of my win time is:  2 weeks 2 days.  
The last win was: 4/12/2022
I'm running version 1.4.0 on ubuntu, with the cli, not the GUI.  

No unexpected errors/warnings in log files (had less than five excessive search time +2sec)
average search for 2905 plots:
 Time: 0.18139 s. - 0.2156 s
My avg eligible plots per:  5.671 
over the last 235258 chances.  

chia farm summary: 

Farming status: Farming
Total chia farmed: <10+>
User transaction fees: 0.000756859156
Block rewards:<10+>
Last height farmed: 182xxxx
Local Harvester
   2905 plots of size: 287.515 TiB
Plot count for all harvesters: 2905
Total size of plots: 287.515 TiB
Estimated network space: 21.932 EiB
Expected time to win: 2 weeks and 2 days

I understand the expected time to win is not an exact number but probability.

**Comment by loppefaaret** [comment]
if we say your estimate is to win twice per month, and 4 months, that is an 8x over ettw. although that is unlucky, it is known to happen from time to time

**Reply by baerrs** [reply]
Thank you for your reply... I ended up rebooting my system, just in case something was weird on my side.  I don't know if my bad luck ended (for this part of my life) or the reboot worked, but I'm winning again.

**Reply by Mogera** [reply]
If you want to stay solo easy thing to fix is to tail your debug.log and grep for 'eligible'.
This will verify incoming challenges, number of plots online and indicate lookup times.
Can also investigate installing Chiadog.

Best answer is .... save yourself the hassle and join a pool as this will almost instantly verify whether your setup is running as intended.
There is no reason not to join a pool as they are reliable, auditable and some of them have voluntary fees (Spacefarmers for example).

Happy farming!
